.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 sudo: false
-dist: trusty
+dist: xenial
```

Comment:
Is this valid?  I'm under the impression that the only valid values here are `precise` or `trusty`

Reply:
well, it looks like it works, even if it isn't in the documentation.

Reply:
It was a shot in the dark after reading https://github.com/travis-ci/travis-ci/issues/8836#issuecomment-356362524. Unfortunately, that didn't fix the build.